db/products.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+let products = [
+  {
+    name: "Supreme x LV Leather Baseball Jacket (Red) ",
+    price: "12000",
+    inventory: "3",
+    id: "1"
+  },
+  {
+    name: "Nike MAG Back to the Future (2016)",
+    price: "36254",
+    inventory: "8",
+    id: "2"
+  },
+  {
+    name: "Rolex Daytona 116595RBOW",
+    price: "157000",
+    inventory: "4",
+    id: "3"
+  }
+];
+
+function getProducts() {
+  return products;
+}
+
+function getSingleProduct(target) {
+  let elementName = "";
+  products.forEach(element => {
+    if (element.id == target) {
+      elementName = element;
+    }
+  });
+
+  return elementName;
+}
+
+function addProduct(name, price, inventory) {
+  let newID;
+
+  for (let i = 0; i < products.length; i++) {
+    newID = (parseInt(products[i].id) + 1).toString();
+  }
```

Comment:
It's easier to just have a integer in this file that keeps track of the current highest id. And just increment it everytime you make a new post, this way you don't reuse ids.